resources/sdk/purecloudgo/templates/configuration.mustache

```diff
@@ -60,6 +61,17 @@ const (
 	USEast2     = "https://api.use2.us-gov-pure.cloud"
 )
 
+func SetPatchConfiguration(patchConfiguration *PatchConfiguration) {
+	instance.PatchConfiguration = &PatchConfiguration{RemoveOmitEmpty: patchConfiguration.RemoveOmitEmpty, APIs: patchConfiguration.APIs}
+	fmt.Println("PatchConfiguration set")
+	fmt.Printf("%#v\n", instance.PatchConfiguration)
+} 
+
+type PatchConfiguration struct {
+	RemoveOmitEmpty bool     `json:"removeOmitEmpty,omitempty"`
```

Comment:
Wondering if this is redundant.  You can already determine if you need to remove a field by whether or not the array has any values.  If the display is empty then you do have not anything to omit.  Removing this field would. make the if condition when determining if you want to remove it much easier to carry out.